config-example.yaml

```
@@ -173,6 +173,7 @@ tls_cert_path: ""
 tls_key_path: ""
 
 log_level: info
+json_logs: false
```

I think this should probably be something like:

```suggestion
log:
  format: json
```

Allowing us to expand in the future.

Level should be migrated to under that root, but that might be a future thing

---

I can migrate level as well, but this is a backward incompatible change, so we should be careful about it. Is there a guideline for these kinds of changes?

---

We are still in a phase were we can afford performing not-that-breaking changes :)

---

Amazing!
I'm sorry for disappearing, was really busy these days. Hopefully will finish this MR this week.